proto/src/universe.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+package universe;
+
+service Universe {
+    rpc CreateKeyspace (CreateKeyspaceRequest) returns (CreateKeyspaceResponse);
+    rpc ListKeyspaces (ListKeyspacesRequest) returns (ListKeyspacesResponse);
+}
+
+message Region {
+    string cloud = 1;
+    string name = 2;
+}
+
+message Zone {
+    Region region = 1;
+    string name = 2;
+}
+
+message CreateKeyspaceRequest {
+    string namespace = 1;
```

Comment:
What should be the mental model of `namespace`? Are they completely isolated from each other in terms of workloads? Maybe the only supported cross-namespace operation is replication?

Reply:
I honestly just added it to provide a way of avoid naming collisions of keyspaces, haven't really thought much more about the uses.